src/expr/src/relation/func.rs

```diff
@@ -1252,45 +1252,88 @@ impl TableFunc {
     }
 
     pub fn output_type(&self) -> RelationType {
-        RelationType::new(match self {
-            TableFunc::JsonbEach { stringify: true } => vec![
-                ScalarType::String.nullable(false),
-                ScalarType::String.nullable(true),
-            ],
-            TableFunc::JsonbEach { stringify: false } => vec![
-                ScalarType::String.nullable(false),
-                ScalarType::Jsonb.nullable(false),
-            ],
-            TableFunc::JsonbObjectKeys => vec![ScalarType::String.nullable(false)],
+        match self {
```

Comment:
I actually prefer the function to be organized like this or something similar to this:
```
pub fn output_type(&self) -> RelationType {
  let (column_types, keys) = match self {
    <stuff>
  };
  let result = RelationType::new(column_types);
  if !keys.is_empty(){
     result = result.with_keys(keys);
  }
  result
}
```
to force people who add future table functions to think about whether output columns for those table functions are unique.